app/controllers/training_and_support_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class TrainingAndSupportController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @provider = current_user.core_induction_programme
+
+    if @provider.blank?
+      raise Pundit::NotAuthorizedError, "This page is only available to Early Career Teachers"
```

Comment:
Not really sure what to do here but the only way it can be accessed is through the URL directly so I'm not sure it's worthy of a dedicated page

Reply:
Well, we certainly don't have designs for that page, so this is good enough for now I think.